### PR TITLE
[TASK] Run db migrations during deployment

### PR DESCRIPTION
**Pull request by wilsonianb**
DEC-480

### CODE REVIEW

**Comment by wilsonianb** [comment]
This won't work. Either the command would need to be run in the container or we would need a copy of the repo outside the container.

**Comment by wilsonianb** [comment]
`db:migrate` is now run by sails bootstrap

**Comment by stevenzeiler** [comment]
Oooh fancy

File: config/bootstrap.js [hunk 1]
```diff
@@ -13,5 +13,9 @@ module.exports.bootstrap = function(cb) {
 
   // It's very important to trigger this callback method when you are finished
   // with the bootstrap!  (otherwise your server will never lift, since it's waiting on the bootstrap)
-  cb();
+  Promise.all([
```
Comment (stevenzeiler) [inline config/bootstrap.js]: Did you use an array here in preparation for more bootstrap functions in the future?

Comment (wilsonianb) [inline config/bootstrap.js]: Yeah, the one bootstrap example I found did that.

Comment (wilsonianb) [inline config/bootstrap.js]: https://gist.github.com/juanpasolano/5c7596d8629eeeb8debd#file-config-models-js

**Comment by stevenzeiler** [comment]
:+1:

**Comment by wilsonianb** [comment]
:+1:

**Pull request merged**